pinot-broker/src/main/java/org/apache/pinot/broker/pruner/PartitionZKMetadataPruner.java

```diff
@@ -26,7 +26,7 @@
 import org.apache.pinot.common.metadata.segment.SegmentZKMetadata;
 import org.apache.pinot.common.request.FilterOperator;
 import org.apache.pinot.common.utils.request.FilterQueryTree;
-import org.apache.pinot.core.data.partition.PartitionFunction;
+import org.apache.pinot.query.spi.PartitionFunction;
```

Comment:
We should re-order the imports for all the changed files. You can do it in IDE in one-step by selecting all the changed local files under `Git` panel and use `optimize imports`

Reply:
There are lots of files are not formatted properly, engineer often forget to format it before commit. How about let's setup a Github workflow, or a maven plugin to ensure all file get formatted:

CLI tools: https://github.com/google/google-java-format
Maven plugins:
https://github.com/diffplug/spotless/tree/main/plugin-maven
https://github.com/coveooss/fmt-maven-plugin
https://github.com/talios/googleformatter-maven-plugin
https://github.com/Cosium/git-code-format-maven-plugin

Reply:
@Jackie-Jiang There are specific requests to not make any `OCD` changes that are not related to refactor. I am fine to make that change if optimizing imports is allowed cc: @mcvsubbu

Reply:
I'd rather leave formatting out of this huge refactor @mqliang. We should definitely set it up outside the scope of this PR though.

Reply:
Sure. Reformatting all files will do a huge disruption to the code base. Let's do it separately, create a issue to track it: https://github.com/apache/incubator-pinot/issues/6727

Reply:
Oh, didn't notice you already created an issue for it. Thanks!

Reply:
@Jackie-Jiang Ok, I'll do that before merging, so that it is easier to review and does not flag reordering based changes.